snow/engine/snowman/transitive.go

```diff
@@ -1063,10 +1045,6 @@ func (t *Transitive) pendingContains(blkID ids.ID) bool {
 	return ok
 }
 
-func (t *Transitive) removeFromPending(blk snowman.Block) {
-	delete(t.pending, blk.ID())
-}
```

Comment:
1 line functions are annoying imo. Also I have no clue why this ever took in the full block rather than just the ID.